doc/crypto/api/ops/signature.rst

```diff
@@ -502,41 +504,73 @@ EdDSA signature algorithms
+    This calculates the Ed25519ctx algorithm as specified in :RFC-title:`8032#5.1`, and requires an Edwards25519 curve key. The `psa_sign_message()` and `psa_verify_message()` functions use an zero-length context when computing or verifying signatures. 
     
-    To use a non-empty context, use the signature functions that accept a context parameter, such as :code:`psa_sign_message_with_context()` and :code:`psa_verify_message_with_context()`
+    To use a non-zero-length context, use the signature functions that accept a context parameter, such as :code:`psa_sign_message_with_context()` and :code:`psa_verify_message_with_context()`
 
     .. admonition:: Implementation note
 
-       Even if you do supply an empty context, signatures created with Ed25519ctx are distinct from those created with PureEdDSA.
+       Even if you supply an zero-length context, signatures created with Ed25519ctx are distinct from those created with PureEdDSA.
 
     .. subsection:: Usage
 
         This is a message signing algorithm. To calculate a signature, use one of the following approaches:
 
         *   Call `psa_sign_message()` or `psa_sign_message_with_context()` with the message.
 
-        Verifying a signature is similar, using, for example, `psa_verify_message()` or `psa_verify_message_with_contex()` instead of the signature function.
+        Verifying a signature is similar, using, for example, `psa_verify_message()` or `psa_verify_message_with_context()` instead of the signature function.
 
     .. subsection:: Compatible key types
 
         | :code:`PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY_TWISTED_EDWARDS)`
         | :code:`PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_TWISTED_EDWARDS)` (signature verification only)
 
 
-.. macro:: PSA_ALG_ED25519PH
+.. macro:: PSA_ALG_ED448CTX
     :definition: ((psa_algorithm_t) 0x0600090B)
 
+    .. summary::
+        Edwards-curve digital signature algorithm with context, using the Edwards448 curve.
+
+        .. versionadded:: 1.4
+        
+    This signature algorithm can be used with both the message and message with context  signature functions.
+
+    This calculates the Ed448ctx algorithm as specified in :RFC-title:`8032#5.1`, and requires an Edwards448 curve key. The `psa_sign_message()` and `psa_verify_message()` functions use an zero-length context when computing or verifying signatures. 
+    
+    To use a non-zero-length context, use the signature functions that accept a context parameter, such as :code:`psa_sign_message_with_context()` and :code:`psa_verify_message_with_context()`
+
+    .. admonition:: Implementation note
+
+       Even if you supply an zero-length context, signatures created with Ed448ctx are distinct from those created with PureEdDSA.
```

Comment:
Ed448ctx does not exist - this is a cut-n-paste error from the ED25519CTX algorithm. The discrepancy is between `PSA_ALG_PURE_EDDSA` and `PSA_ALG_EDDSA_CTX` for 255-bit keys.